.metadata/idl/state.idl

```diff
@@ -1,6 +1,20 @@
 import "objidl.idl";
 import "oaidl.idl";
 import "propidl.idl";
+// import "wtypes.idl";
+
+typedef struct StateData
```

Comment:
If this struct is removed, and getbool() returns BOOLEAN, bindgen can correctly reference windows crate BOOLEAN type.